C_routing.md

```diff
@@ -649,17 +628,17 @@ Older versions of Phoenix defined a third pipeline `:before`. Its purpose was to
 
 Phoenix defines two other pipelines by default, `:browser` and `:api`. The router will invoke these after it matches a route, assuming we have called `pipe_through/1` with them in the enclosing scope.
 
-As their names suggest, the `:browser` pipeline prepares for routes which render HTML for a browser. The `:api` pipeline prepares for routes which produce data for an api.
+As their names suggest, the `:browser` pipeline prepares for routes which render HTML for a browser. The `:api` pipeline prepares for routes which produce data for an api, for example JSON.
 
 The `:browser` pipeline has four plugs: `plug :accepts, ~w(html)` which defines the request format or formats which will be accepted, `:fetch_session`, which, naturally, fetches the session data and makes it available in the connection, `:fetch_flash` which retrieves any flash messages which may have been set, and  `:protect_from_forgery`, which protects form posts from cross site forgery.
 
-Currently, the `:api` pipeline only defines `plug :accepts, ~w(json)`.
+Currently our example so far, the `:api` pipeline only defines `plug :accepts, ~w(json)`.
```

Comment:
Actually, the idea is that currently Phoenix only defines the `plug :accepts, ~w(json)`, not that our example does. 

Please remove "our example so far".